Compiler/compilerLib.py

```diff
@@ -206,6 +206,12 @@ def build_option_parser(self):
             dest="edabit",
             help="mixing arithmetic and binary computation using edaBits",
         )
+        parser.add_option(
```

Comment:
I tried setting the flag for comparison_rabbit with `program.use_comparison_rabbit(True)` in the .mpc file (similar to `use_edabits`), but it always remained false, and I only managed to set it true by adding `--comparison_rabit` in the compilation instruction. Do you see what I am doing wrong there?